runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayUtil.java

```diff
@@ -181,6 +181,43 @@ public static void renderHoverableArea(Graphics2D graphics, Area area, net.runel
 		}
 	}
 
+	public static void renderNPCOverlay(Graphics2D graphics, Client client, NPC npc, Color color, HighlightStyle style, boolean drawNames)
```

Comment:
Cant this simply work for all actors? Not just NPCs

Reply:
And maybe drawNames do not need to be here and the logic for drawing them can stay from where it originated from